### PR TITLE
[WIFI-11196] Restructure repository into multimodule project

### PR DESCRIPTION
**Pull request by elludraon**
Restructure the repository from a single-module Maven project to a multi-module Reactor project. This mainly splits the boilerplate code related to the OpenWiFi CloudSDK out of the base project.

Changes:
- Most original directory contents moved into `owrrm` subdirectory
- CloudSDK specific code moved to new subdirectory `lib-cloudsdk`
- New root (parent) directory declares submodules `owrrm`, `lib-cloudsdk` as a Reactor project
- Package `com.facebook.openwifirrm` moved to `com.facebook.openwifi.rrm`
- Subpackage `ucentral` moved under `cloudsdk` and refactored
- Javadoc build has changed from `mvn javadoc:javadoc` to `mvn javadoc:aggregate`

Minor notes:
- `UCentralUtils.channelToFrequencyMHz()` implementation is simplified to avoid dependency on `ChannelOptimizer.AVAILABLE_CHANNELS_BAND`
- `UCentralUtils.generateServiceKey()` is moved to `Utils` to avoid dependency on `RRMConfig.ServiceConfig`
- `UCentralClient` constructor parameter `UCentralSocketParams` is replaced with individual members to avoid dependency on `RRMConfig`
- `AggregatedState` is moved upwards out of the `models` subpackage
- Fixed various Javadoc errors in existing code (`ModelerUtils`)

### CODE REVIEW

File: lib-cloudsdk/pom.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <finalName>openwifi-cloudsdk</finalName>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <includes>
+          <include>**/*</include>
+        </includes>
+      </resource>
+    </resources>
+    <plugins>
```
Comment (RockyMandayam2) [inline lib-cloudsdk/pom.xml:25]: Is it possible to / Is it recommended to take the common plugins and move them up into the higher level pom?

**Review by RockyMandayam2 — APPROVED** [review]
Thanks! Just left one comment.

**Review by jwoos — APPROVED** [review]
if it builds, it ships

File: lib-cloudsdk/pom.xml (hunk 1 above)
Comment (elludraon) [reply lib-cloudsdk/pom.xml:25]: They are already in the parent pom, that's the `pluginManagement` section (and similarly `dependencyManagement`), hence why there are only stubs in the child pom.

**Pull request merged**